login-workflow/package.json

```diff
@@ -23,7 +23,7 @@
     },
     "prettier": "@pxblue/prettier-config",
     "dependencies": {
-        "@pxblue/react-auth-shared": "^3.0.0",
+        "@pxblue/react-auth-shared": "^3.1.0",
```

Comment:
this'll be 3.2.0 to pick up your update type definition that you need. That will be published as soon as this PR gets another approval:
https://github.com/pxblue/react-auth-shared/pull/25